### PR TITLE
Fixes search icon disappearance

### LINKED ISSUE #1042: Search button disappears when search option in action menu bar is selected
**Describe the bug**
In Bookmark/History page, when a user presses search button present in action bar, and types the text followed by selecting Search option from the action menu, search button disappears upon pressing back arrow button or closing search bar.

**Expected behavior**
Search option should be still present after pressing back arrow button in action bar.

**Steps to reproduce the behavior:**
1. Open Bookmark/History page.
2. Press search icon and type some text.
3. Select the search option from the action menu (three dots in action bar).
4. Press the back arrow button or cross button. The search icon and search option in the action menu both disappear.

**Screenshots**
search_disappears

**Environment**
- Version of Kiwix Android : Build 55
- Device : Redmi 4
- OS version : Android Nougat 7.1

### PR DESCRIPTION
**Pull request by sonusourav**
Fixes #1042 

Changes: 
1. Changed showAsAction property of search menu item to **always** instead of ifRoom.
*app:showAsAction="always|collapseActionView"*
2. Search option is removed from overflow menu (three dots) when actionView mode is on (i.e. when search bar is visible).

Screenshots/GIF for the change: 
search_menu_item

### CODE REVIEW

**Comment by sonusourav** [comment]
@mhutti1 Please review

**Comment by sonusourav** [comment]
@abdulwd Could you please review this.

File: app/src/main/res/menu/menu_bookmarks.xml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
```
Comment (abdulwd) [inline app/src/main/res/menu/menu_bookmarks.xml]: Reformat using Square's code style `tools` will be below `app`.

Comment (sonusourav) [reply app/src/main/res/menu/menu_bookmarks.xml]: @abdulwd Changes done. Please review.

**Comment by abdulwd** [comment]
:+1: for taking care of the lint warnings.

**Pull request merged**